packages/components/pages/f-checkout/src/components/Notes.vue

```diff
@@ -69,10 +69,7 @@ export default {
 
     computed: {
         ...mapState(VUEX_CHECKOUT_MODULE, [
-            'features',
-            'notesConfiguration',
-            'serviceType',
```

Comment:
Looks like `serviceType` is still used in the template, right?

Reply:
Yep, I got too trigger happy and accidentally deleted that as well late on a Friday. 😅